webui/react/src/types.ts

```diff
@@ -367,6 +367,28 @@ export type HyperparameterType = t.TypeOf<typeof HyperparametersType>;
 export type Hyperparameters = {
   [keys: string]: Hyperparameters | HyperparameterBase;
 };
+
+export const PachdIntegration = t.type({
```

Comment:
the structure here implies that the integration structure may include more integrations in the future. as such, the `pachyderm` prop should be optional, and the name of this should reflect that.